packages/slice-machine/src/legacy/lib/models/common/widgets/Link/components.tsx

```diff
@@ -95,3 +89,41 @@ export function RepeatableCheckbox(props: CommonCheckboxProps) {
     </Box>
   );
 }
+
+export function Variant() {
+  return (
+    <Box overflow="hidden" flexDirection="column" border borderRadius={6}>
+      <Box
+        border={{ bottom: true }}
+        padding={12}
+        flexDirection="column"
+        gap={8}
+      >
+        <Text variant="h4" color="grey12">
+          Variants
+        </Text>
+        <Text color="grey11">
+          Add variants like "Primary" and "Secondary" to allow editors to choose
+          the link's style by selecting one of them.
+        </Text>
+        <Box gap={8}>
+          <Switch />
+          <Text color="grey11">Disabled</Text>
```

Comment:
⚠️ If I get the Figma, I think the text should change depending on whether it's checked or not.

If your plan was to do it later on, forget my comment.

Reply:
yup, this was just putting the stuff there with no behavior